script/conflate_tiger.sh

```diff
@@ -56,8 +56,9 @@ if [ ! -d "$TIGERPATH/shapefiles" ]; then
   exit 1;
 fi
 
-# recurse through filesystem listing all .shp file names
-find "$TIGERPATH/shapefiles" -type f -iname "*.shp" -print0 |\
+# recurse through filesystem listing all .shx file names
+# some county zip packages are missing .shx which causes the ogr2ogr script to fail
+find "$TIGERPATH/shapefiles" -type f -iname "*.shx" -print0 |\
```

Comment:
I mentioned the reason for the change in the comment:
`some county zip packages are missing .shx which causes the ogr2ogr script to fail`
I was hitting errors when running the scripts and it seemed that there were some files that didn't have the `*.shx` file and that was causing the build to choke. Looking just for the `*.shx` simply avoids the problematic ones.